storage/blockdevice/tests/UNITTESTS/SFDP/test_sfdp.cpp

```diff
@@ -17,8 +17,43 @@
 #include "gtest/gtest.h"
 #include "gmock/gmock.h"
 #include "blockdevice/internal/SFDP.h"
+#include "test_data.h"
```

Comment:
I'd prefer the test data next to the tests that use these, or at least just within the same cpp file. I don't think the indirection is worthwhile.

Reply:
I separated it only because the arrays are big and take up many lines. Logically it should be in the same cpp file, and some other unit test files are pretty big too, so I will remove the header.